tests/playwright/src/specs/kubernetes.spec.ts

```diff
@@ -126,16 +129,103 @@ test.describe('Kubernetes resources End-to-End test', () => {
       const pvcDetails = await pvcsPage.openResourceDetails(PVC_NAME, KubernetesResources.PVCs);
       await playExpect(pvcDetails.heading).toBeVisible();
       await playExpect
-        .poll(async () => pvcDetails.getState(), { timeout: 40_000 })
+        .poll(async () => pvcDetails.getState(), { timeout: 50_000 })
         .toEqual(KubernetesResourceState.Running);
     });
     test('Delete the PVC resource', async ({ page, navigationBar }) => {
-      await deletePod(page, POD_NAME);
+      await deletePod(page, PVC_POD_NAME);
       const kubernetesBar = await navigationBar.openKubernetes();
       const pvcsPage = await kubernetesBar.openTabPage(KubernetesResources.PVCs);
       await pvcsPage.deleteKubernetesResource(PVC_NAME);
       await handleConfirmationDialog(page);
       await playExpect(pvcsPage.getResourceRowByName(PVC_NAME)).not.toBeVisible();
     });
   });
+  test.describe.serial('ConfigMaps and Secrets lifecycle test', () => {
+    test('Create ConfigMap resource', async ({ navigationBar }) => {
+      const podsPage = await navigationBar.openPods();
+      await playExpect(podsPage.heading).toBeVisible();
+      const playYamlPage = await podsPage.openPlayKubeYaml();
+      await playExpect(playYamlPage.heading).toBeVisible();
+      const configmapYamlFilePath = path.resolve(
+        __dirname,
+        '..',
+        '..',
+        'resources',
+        'kubernetes',
+        `${CONFIGMAP_NAME}.yaml`,
+      );
+      await playYamlPage.playYaml(configmapYamlFilePath, {
+        runtime: PlayYamlRuntime.Kubernetes,
+        kubernetesContext: KUBERNETES_CONTEXT,
+        kubernetesNamespace: KUBERNETES_NAMESPACE,
+      });
```

Comment:
Can you please extract the object into constant since it is called multiple times?